umm-spec-lib/src/cmr/umm_spec/test/iso_shared.clj

```diff
@@ -0,0 +1,56 @@
+(ns cmr.umm-spec.test.iso-shared
+  "ISO 19115 specific expected conversion functionality"
+  (:require
+    [clojure.string :as string]
+    [cmr.common.util :as util :refer [update-in-each]]
+    [cmr.umm-spec.iso19115-2-util :as iso-util]
+    [cmr.umm-spec.models.umm-collection-models :as umm-c]
+    [cmr.umm-spec.models.umm-common-models :as cmn]
+    [cmr.umm-spec.util :as su]
+    [cmr.umm-spec.xml-to-umm-mappings.iso-shared.iso-topic-categories :as iso-topic-categories])
+  (:use
+   [cmr.umm-spec.models.umm-collection-models]
+   [cmr.umm-spec.models.umm-common-models]))
+
+(defn split-temporals
+  "Returns a seq of temporal extents with a new extent for each value under key
+  k (e.g. :RangeDateTimes) in each source temporal extent."
+  [k temporal-extents]
+  (reduce (fn [result extent]
+            (if-let [values (get extent k)]
+              (concat result (map #(assoc extent k [%])
+                                  values))
+              (concat result [extent])))
```

Comment:
This produces different results and the tests fail.  Here is the difference between concat and conj 

user=> (concat [1 2 3] [4 5 6])
(1 2 3 4 5 6)

user=> (conj [1 2 3] [4 5 6])
 [1 2 3 [4 5 6]]